rpmbuild.sh

```diff
@@ -2,6 +2,7 @@
 rm -rf dist postprocessing.egg-info
 python3 -m build --sdist
 cp dist/postprocessing-*.tar.gz ~/rpmbuild/SOURCES/
+cp -r systemd ~/rpmbuild/SOURCES/
```

Comment:
Please add a comment that the file isn't included in the sdist.

Also, this is kinda dangerous. `~/rpmbuild/SOURCES/` gets full of all sorts of random junk and creating a directory called `systemd/` is likely to clash with another package. What if instead this
```
cp systemd/autoreduce-queue-processor.service ~/rpmbuild/SOURCES/
```
and adjust the spec file? This is a very unique name already and makes it easier to see/delete from a package maintainers `~/rpmbuild/` tree.